pennylane/shadows/classical_shadow.py

```diff
@@ -390,16 +391,17 @@ def max_entangled_circuit():
 
             @qml.qnode(dev)
             def qnode(x):
-                for i in range(n_wires):
+                for i in range(wires):
+
                     qml.RY(x[i], wires=i)
```

Comment:
I cannot run this example as well because I don't have the right device and the array x, @eddddddy @Qottmann

Reply:
Not sure what values @Qottmann was using. If needed we can just redefine `x` and change the output below.